cmd/nfd-master/main.go

```diff
@@ -65,7 +65,7 @@ func argsParse(argv []string) (master.Args, error) {
   Usage:
   %s [--no-publish] [--label-whitelist=<pattern>] [--port=<port>]
      [--ca-file=<path>] [--cert-file=<path>] [--key-file=<path>]
-     [--verify-node-name] [--extra-label-ns=<list>]
+     [--verify-node-name] [--extra-label-ns=<list>] [--ext-res=<list>]
```

Comment:
Since the feature in k8s is "extended resource" and adding one has an impact on multiple places inside the node status, not just capacity, I wouldn't go there. But yes, I agree that --ext-res is not perhaps the best either. This may require a bit more time to come up with something good.

Reply:
I don't have strong vision about the name of the flag. My thinking behind capacity was that our interface is the node capacity attribute, and, we're not doing any resource management, just announcing the capacity.

Reply:
In case longer argument names are not frowned upon, I propose "--resource-labels"